samples/extensions/extended_dynamic_state2/README.md

```diff
@@ -42,336 +42,336 @@ Previously developers had to create multiple pipelines for different parameters
-	graphics_create.pVertexInputState   = &vertex_input_state;
-	graphics_create.pTessellationState  = VK_NULL_HANDLE;
-	graphics_create.stageCount          = 2;
-	graphics_create.pStages             = shader_stages.data();
-	graphics_create.layout              = pipeline_layouts.baseline;
+VkPipelineInputAssemblyStateCreateInfo input_assembly_state =
+	vkb::initializers::pipeline_input_assembly_state_create_info(
+	    VK_PRIMITIVE_TOPOLOGY_TRIANGLE_LIST,
+	    0,
+	    VK_FALSE);
+
+VkPipelineRasterizationStateCreateInfo rasterization_state =
+	vkb::initializers::pipeline_rasterization_state_create_info(
+	    VK_POLYGON_MODE_FILL,
+	    VK_CULL_MODE_BACK_BIT,
+	    VK_FRONT_FACE_CLOCKWISE,
+	    0);
+
+rasterization_state.depthBiasConstantFactor = 1.0f;
+rasterization_state.depthBiasSlopeFactor    = 1.0f;
+rasterization_state.depthBiasClamp          = 0.0f;
+
+VkPipelineColorBlendAttachmentState blend_attachment_state =
+	vkb::initializers::pipeline_color_blend_attachment_state(
+	    VK_COLOR_COMPONENT_R_BIT | VK_COLOR_COMPONENT_G_BIT | VK_COLOR_COMPONENT_B_BIT | VK_COLOR_COMPONENT_A_BIT,
+	    VK_TRUE);
+
+blend_attachment_state.colorBlendOp        = VK_BLEND_OP_ADD;
+blend_attachment_state.srcColorBlendFactor = VK_BLEND_FACTOR_SRC_ALPHA;
+blend_attachment_state.dstColorBlendFactor = VK_BLEND_FACTOR_ONE_MINUS_SRC_ALPHA;
+blend_attachment_state.alphaBlendOp        = VK_BLEND_OP_ADD;
+blend_attachment_state.srcAlphaBlendFactor = VK_BLEND_FACTOR_ONE;
+blend_attachment_state.dstAlphaBlendFactor = VK_BLEND_FACTOR_ZERO;
+
+VkPipelineColorBlendStateCreateInfo color_blend_state =
+	vkb::initializers::pipeline_color_blend_state_create_info(
+	    1,
+	    &blend_attachment_state);
+
+/* Note: Using Reversed depth-buffer for increased precision, so Greater depth values are kept */
+VkPipelineDepthStencilStateCreateInfo depth_stencil_state =
+	vkb::initializers::pipeline_depth_stencil_state_create_info(
+	    VK_TRUE, /* changed */
+	    VK_TRUE,
+	    VK_COMPARE_OP_GREATER);
+
+VkPipelineViewportStateCreateInfo viewport_state =
+	vkb::initializers::pipeline_viewport_state_create_info(1, 1, 0);
+
+VkPipelineMultisampleStateCreateInfo multisample_state =
+	vkb::initializers::pipeline_multisample_state_create_info(
+	    VK_SAMPLE_COUNT_1_BIT,
+	    0);
+
+VkPipelineTessellationStateCreateInfo tessellation_state =
+	vkb::initializers::pipeline_tessellation_state_create_info(3);
+
+
+std::vector<VkDynamicState> dynamic_state_enables = {
+	VK_DYNAMIC_STATE_VIEWPORT,
+	VK_DYNAMIC_STATE_SCISSOR,
+	VK_DYNAMIC_STATE_PRIMITIVE_TOPOLOGY_EXT,
+	VK_DYNAMIC_STATE_DEPTH_BIAS_ENABLE_EXT,
+	VK_DYNAMIC_STATE_RASTERIZER_DISCARD_ENABLE_EXT,
+	VK_DYNAMIC_STATE_PRIMITIVE_RESTART_ENABLE_EXT,
+};
+VkPipelineDynamicStateCreateInfo dynamic_state =
+	vkb::initializers::pipeline_dynamic_state_create_info(
+	    dynamic_state_enables.data(),
+	    static_cast<uint32_t>(dynamic_state_enables.size()),
+	    0);
+
+// Vertex bindings an attributes for model rendering
```

Comment:
I think it should be:
// Vertex bindings and attributes for model rendering

Reply:
fixed, the same change added in .cpp file